### PR TITLE
Bps 765 site config

### PR DESCRIPTION
**Pull request by mdayican**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/BPS-765

### Change description ###

site config added- helm setting and bootstrap added

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: charts/bulk-scan-payment-processor/values.yaml [hunk 1]
```diff
@@ -3,3 +3,10 @@ java:
   ingressHost: bulk-scan-payment-processor-{{ .Values.global.environment }}.service.core-compute-{{ .Values.global.environment }}.internal
   environment:
   image: 'https://hmctspublic.azurecr.io/bulk-scan/payment-processor:latest'
+  keyVaults:
+    "bulk-scan":
+      resourceGroup: payment-processor
```
Comment (nitinprabhu) [inline charts/bulk-scan-payment-processor/values.yaml]: this should be `bulk-scan` ?

File: src/main/resources/application.yaml [hunk 2]
```diff
@@ -16,3 +16,16 @@ management:
 spring:
   application:
     name: Bulk Scan Payment Processor
+
+site-configuration:
```
Comment (nitinprabhu) [inline src/main/resources/application.yaml]: we can just say `site-mappings` ?

File: charts/bulk-scan-payment-processor/values.yaml [hunk 3]
```diff
@@ -3,3 +3,10 @@ java:
...
+      secrets:
+        - site-id-probate
+        - site-id-divorce
+        - site-id-finrem
```
Comment (doncem) [inline charts/bulk-scan-payment-processor/values.yaml]: you will have to create those secrets manually

File: charts/bulk-scan-payment-processor/values.yaml (hunk 1 above)
Comment (mdayican) [reply charts/bulk-scan-payment-processor/values.yaml]: fixed

File: src/main/resources/application.yaml (hunk 2 above)
Comment (mdayican) [reply src/main/resources/application.yaml]: fixed

File: src/test/java/uk/gov/hmcts/reform/bulkscan.payment.processor/controllers/RootControllerTest.java [hunk 4]
```diff
@@ -2,14 +2,17 @@
 
 import org.junit.jupiter.api.DisplayName;
 import org.junit.jupiter.api.Test;
+import org.junit.runner.RunWith;
```
[the remark this answers is not in the document]
Comment (mdayican) [reply src/test/java/uk/gov/hmcts/reform/bulkscan.payment.processor/controllers/RootControllerTest.java]: fixed

File: charts/bulk-scan-payment-processor/values.yaml (hunk 3 above)
Comment (mdayican) [reply charts/bulk-scan-payment-processor/values.yaml]: ok

**Pull request merged**